app/models.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, first_name, last_name, username, password):
         self.password = password
 
     def generate_auth_token(self, expiration=100):
-        from views import app
+        from app.views import app

```

Comment:
Please do your imports outside the methods. At the top of the page

Reply:
ok done